modules/ROOT/pages/indexes-for-vector-search.adoc

```diff
@@ -87,14 +88,46 @@ The index name must be unique among both indexes and constraints.
 ====
 
 A vector index is a single-label, single-property index for nodes.
-In addition to the label and property key (both given as `STRING`), a vector index needs to be configured with both the dimensionality of the vector (`INTEGER` between `1` and `2048` _inclusive_), and the measure of similarity between two vectors (case-insensitive `STRING`).
+A vector index needs to be configured with both the dimensionality of the vector (`INTEGER` between `1` and `2048` _inclusive_), and the measure of similarity between two vectors (case-insensitive `STRING`).
 For details, see xref:#indexes-vector-similarity[].
 
-.Signature for `db.index.vector.createNodeIndex` to create a vector node index
-[source,syntax,role="noheader",indent=0]
+[NOTE]
+====
+More details about the syntax descriptions can be found link:{neo4j-docs-base-uri}/operations-manual/{page-version}/database-administration/syntax/#administration-syntax-reading[here].
+====
+
+.Syntax for creating vector indexes
+[options="header", width="100%", cols="5a, 3"]
+|===
+| Command | Description
+
+| [source, syntax, role=noplay, indent=0]
 ----
-db.index.vector.createNodeIndex(indexName :: STRING, label :: STRING, propertyKey :: STRING, vectorDimension :: INTEGER, vectorSimilarityFunction :: STRING)
+CREATE VECTOR INDEX [index_name] [IF NOT EXISTS]
+FOR (n:LabelName)
+ON (n.propertyName)
+OPTIONS "{" option: value[, ...] "}"
 ----
+| Create a vector index on nodes.
+The options map is mandatory since there are mandatory index configuration settings, the vector dimensions and similarity function.
```

Comment:
Consider putting the mandatory options into the example syntax

Reply:
If feels odd to only add the config as it seems to imply that is the only available options, and with only one existing provider it feels odd to add the provider in the syntax description :( This is also the general way we describe options for all indexes/other places that have them so being more explicit here feels odd as well :(